static/js/main.js

```diff
@@ -0,0 +1,85 @@
+$(document).ready(function() {
+
+  google.charts.load("current", {
+    packages: ["calendar"]
+  });
+
+  var frm = $('#username_form');
+  frm.submit(function() {
+    $("#message").html(
+      "<div class='panel panel-default message'><div class='panel-body'>Creating Graphs...</div></div>"
+    );
+    $.ajax({
+      type: frm.attr('method'),
+      data: frm.serialize(),
+      success: function(data) {
+        google.charts.setOnLoadCallback(drawCharts(data));
+      },
+      error: function(data) {
+        $("#message").html(
+          "<div class='panel panel-default message'><div class='panel-body'>Something went wrong!</div></div>"
```

Comment:
You probably want to surface the error here, otherwise you'll get fairly useless error reports from your users :)